README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Kieran Coffey
+I am a MAPSS student studying European History - specifically German cultural and political history of the nineteenth and twentieth centuries. At the moment I am studying the relationship between conceptions of regional, national, and religious identity as they relate to the German concept of *Heimat*, or homeland. My other academic interests include:
+
+    - nationalism and national identity
+    - sub-national or regional identity
+    - mass political movements, particularly socialist/communist and far-right movements
+    - borderland regions, particular between Germany and France 
+    and the European Union 
+
+## Outside of Academia
+I also like to hike, watch soccer, play video games (especially grand stratedgy games! <https://www.paradoxplaza.com>), trying new recipes, and visiting friends in other cities and countries.  
+Originally I'm from Buffalo, New York where I went to highschool and where my family still lives. I went to college at the University of Pittsburgh, and graduated in 2017. I then took language classes at the Goethe Institut in Freiburg im Breisgau, Germany (<https://en.wikipedia.org/wiki/Freiburg_im_Breisgau>) and interned in Stuttgart, Germany at the Heinrich Böll Stiftung (Heinrich Böll Foundation.  
```

Comment:
When embedding links, it's better to use a text link. That is, don't just paste the link but put it in text to be clicked on (as you would see on most websites)

```
[Goethe Institut in Freiburg im Breisgau, Germany](https://en.wikipedia.org/wiki/Freiburg_im_Breisgau)
```

[Goethe Institut in Freiburg im Breisgau, Germany](https://en.wikipedia.org/wiki/Freiburg_im_Breisgau)